heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -452,6 +487,28 @@ public static void setTopologyExactlyOnceEnabled(Map<String, Object> conf, boole
     conf.put(Config.TOPOLOGY_EXACTLYONCE_ENABLED, String.valueOf(exactOnce));
   }
 
+  public static void setTopologyPackingAlgorithm(Map<String, Object> conf,
+                                                 PackingAlgorithmType type) {
```

Comment:
This would cause a problem, since IPacking and IRepacking are in spi, which depends on api, but not the other way around. We could considering moving these interfaces into api. @kramasamy do you have an opinion on this one?